docs/source/user_guide/01a_installation.rst

```diff
@@ -0,0 +1,34 @@
+
+Installation
```

Comment:
How about detailed installation in the docs and perhaps README has quick install (if relevant) and links to the docs. We might be able to have install in the README and generate the doc from it so they would be in sync. Let me think on that.